package.json

```
@@ -18,7 +18,9 @@
     "build": "lerna run build",
     "clean": "lerna run clean",
     "build:tsc": "tsc -b packages/**/tsconfig.build.json",
-    "lint": "eslint . --fix --quiet",
+    "lint": "lerna run lint:check",
```

I think we should run `lerna run lint:check` in the `lint:check` script instead
```suggestion
    "lint": "lerna run lint",
```